ext/standard/tests/strings/chunk_split_variation2_32bit.phpt

```diff
@@ -0,0 +1,25 @@
+--TEST--
+Test chunk_split() function : usage variations - unexpected large '$end' string argument variation 2
+--SKIPIF--
+<?php
+if (PHP_INT_SIZE != 4) die("skip this test is for 32bit platform only");
+?>
+--FILE--
+<?php
+/* Prototype  : string chunk_split(string $str [, int $chunklen [, string $ending]])
+ * Description: Returns split line
+ * Source code: ext/standard/string.c
+ * Alias to functions: none
+*/
+
+echo "*** Testing chunk_split() : unexpected large 'end' string argument variation 2 ***\n";
+
+$a=str_repeat("B", 65537);
+$b=1;
+$c=str_repeat("B", 65537);
+var_dump(chunk_split($a,$b,$c));
+?>
+--EXPECTF--
+*** Testing chunk_split() : unexpected large 'end' string argument variation 2 ***
+
+Fatal error: Possible integer overflow in memory allocation (65538 * 65537 + 65556) in %s on line %d
```

Comment:
IMO throwing a fatal error and using words like "possible" is very confusing to the consumers. If something is (was) _possible_, it doesn't necessarily mean it happened, only that it might.

Reply:
Seems like a lot of Zend's ``zend_safe_*`` functions use this message, so i would expect it to be changed in all possible messages.

See https://lxr.room11.org/xref/php-src%40master/Zend/zend_multiply.h#318 for an example.

Reply:
Agree that it would be good to drop the "possible" from the error message (as an independent change).